docs/hoon/library/1a.md

```diff
@@ -105,35 +129,111 @@ An atom.
 Source
 ------
 
-    ++  div                                                 ::  divide
+    ++  div                                                 
       ~/  %div
-      |=  [a/@ b/@]
+      |:  [a=`@`1 b=`@`1]
       ^-  @
-      ~|  'div'
+      ~_  leaf+"divide-by-zero"
       ?<  =(0 b)
       =+  c=0
       |-
       ?:  (lth a b)  c
       $(a (sub a b), c +(c))
     ::
-
 Examples
 --------
 
     > (div 4 2)
     2
+
     > (div 17 8)
     2
+
     > (div 20 30)
     0
 
+    > (div 'ab' 'a')
+    259
+    > `@`'ab'
+    25.185
+    > `@`a
+    97
+
+    > (div 0x9f 0x29)
+    6
+    > `@`0x9f                                    ::`0x9f` is an atom represented in the hexadecimal soft atom type (`@ux`).
+    159
+    > `@`0x29
+    41
+
+    > (div 0x29 0x9f)
+    0
+
+    > (div ~sibtel-tallyd ~zod)                 
+    ! exit
+    > `@`~sibtel-tallyd
+    7.303.014
+    > `@`~zod                                    ::the above operation returned an error because `~zod` is a representation of `0`.
+    0
+
+    > `@p`2                                      ::this casts `2` to `@p` to produce the ship name which represents that atom.
+    ~bud
+    > (div ~sibtel-tallyd ~bud)
+    > 3.651.507
+
+
+***
+### `++dvr`
+
+Divide with remainder
+
+Computes `a` divided by `b` and the remainder of that operation.
+
+Produces
+-------
+
+A cell of atoms.
+
+Source
+------
+
+  ++  dvr
+    ~/  %dvr
+    |=  [a=@ b=@]     
+    ^-  [p=@ q=@]
+    [(div a b) (mod a b)]
+  ::
+Examples
+--------
+    > (dvr 17 8)
+    [p=2 q=1]
+
+    > (dvr 17 6)
+    [p=2 q=5]
+
+    > (dvr 18 6)
+    [p=3 q=0]
+
+    > (dvr 4 2)
+    [p=2 q=0]
+
+    > (dvr 'ab 'a')
+    [p=259 q=62]
+    > `@`'a'                                     ::`a` is an atom represented as a different data type.
+    97                                             We can cast it to the aura `@` to see what its integer representation is.
+    > `@`'ab'
+    25.185       
+
+    > (dvr '' 'a')
+    [p=0 q=0]
+
 
 ***
 ### `++gte`
 
 Greater-than/equal
 
-Tests whether `a` is greater than a number `b`.
+Tests whether a number `a` is greater than or equal a number `b`.
```

Comment:
"or equal *to*"

We probably shouldn't say "number" here, because this doesn't work for all Hoon "numbers" in general.  (e.g., try `(lth .12 .~11)` in the dojo... it'll return %.y).  Let's just say `a` and `b`, and let the input section specify "atom".